graph/encoding/dot/example_test.go

```diff
@@ -0,0 +1,44 @@
+package dot_test
```

Comment:
Sorry, I missed this. Can you please add a copyright header.

Reply:
resolved